tests/test_compare_objects.py

```diff
@@ -0,0 +1,12 @@
+import tests
+from pyVmomi import vim
+from operator import eq as equals
+
+
+class CompareTests(tests.VCRTestBase):
+
+    def test_compare_objects(self):
+        compare_str = "Test"
+        compare_obj = vim.Datastore
+        self.assertFalse(equals(compare_str, compare_obj))
+        self.assertTrue(equals(compare_obj, compare_obj))
```

Comment:
So the first test should fail in python2 and second should fail in python3. 

def test_compare_object_unicode(self):
     compare_unicode = u"Test"
     compare_obj = vim.Datastore
     self.assertFalse(equals(compare_unicode, compare_obj))

def test_compare_objects_bytes(self):
     compare_bytes = b"Test"
     compare_obj = vim.Datastore
     self.assertFalse(equals(compare_bytes, compare_obj))